src/it/resources/list.yaml

```diff
@@ -18,9 +18,24 @@ objc:
 objcpp:
   translator: '::djinni::List'
   header: '"djinni/objc/DJMarshal+Private.h"'
+java:
+  typename: 'java.util.ArrayList'
+  boxed: 'java.util.ArrayList'
+  reference: true
+  generic: true
+  hash: '%s.hashCode()'
+  writeToParcel: 'out.writeLong(%s.getTime())'
+  readFromParcel: 'new %s(in.readLong())'
+jni:
+  translator: '::djinni::List'
+  header: '"djinni/jni/Marshal.hpp"'
+  typename: jobject
+  typeSignature: 'Ljava/util/ArrayList;'
 cs:
   translator: '::djinni::List'
   header: '"djinni/cppcli/Marshal.hpp"'
   typename: "System::Collections::Generic::List"
   reference: true
   generic: true
+
+# No C# definitions!
```

Comment:
Can you clarify what you mean? The C# definitions are right above the comment if I'm not mistaken?

Reply:
yea..not sure how that got in there.....will remove it.